code/modules/recycling/sortingmachinery.dm

```diff
@@ -458,9 +458,9 @@
 	. = ..()
 	var/potential_cut = input("How much would you like to payout to the registered card?","Percentage Profit") as num|null
 	if(!potential_cut)
-		percent_cut = 50
-	percent_cut = clamp(round(potential_cut, 1), 1, 50)
-	to_chat(user, "<span class='notice'>[percent_cut]% profit will be received if a package with a barcode is sold.</span>")
+		cut_multiplier = 0.2
```

Comment:
doesn't this change behavior? or was it unintentionally set to 50%

Reply:
I made it more consistent. You can still set this to anything between 1% and 50%. In some cases barcode values defaulted to 50% and in others 20%. I thought 20% made more sense to start off at.

Reply:
Actually I've changed it to default to 50% "split" (has never actually affected cargo) and use initial() to set the var if the player doesn't input anything in these cases